src/styles/shiki.css

```
@@ -5,14 +5,18 @@ html .shiki:not(.echo-code) {
 
   @apply !bg-black;
   & span.line {
-    & > span {
-      color: var(--shiki-dark) !important;
-    }
-
     @apply bg-[hsl(240,37%,3%)];
   }
 }
 
+html .shiki:not(.echo-code):not(.mcp-snippet) span.line > span {
```

Stylelint (17.7.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)